src/main/java/page/objects/DistrictTypeDropdownOption.java

```diff
@@ -1,10 +1,11 @@
-package PageObjects;
+package page.objects;
 
 import org.openqa.selenium.By;
 
 public enum DistrictTypeDropdownOption {
     SILESIA_DISTRICT(By.cssSelector("#regions option:nth-child(13)"));
```

Comment:
To samo co https://github.com/KonradSar/PERNWebsiteAutomatedTesting/pull/6/files#r1156471792